styles.css

```diff
@@ -1 +1,46 @@
 /* This will be the styles file for Healthy-Restaurant */
+body {
+  margin: 0 160px;
+  background: black;
+  color: white;
+  font-family: "Poppins", sans-serif;
+}
+header {
+  display: flex;
+  height: 80px;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 161px 20px 147px;
```

Comment:
We should not be using horizontal paddings now as we added that to body, we can use vertical paddings only